### PR TITLE
Restoring old behavior of the outpoint copy to clipboard feature

### PR DESCRIPTION
**Pull request by bilthon**
This fixes the issue described at #5878

### CODE REVIEW

File: electrum/gui/qt/util.py [hunk 1]
```diff
@@ -653,7 +653,11 @@ def add_copy_menu(self, menu, idx):
         for column in self.Columns:
             column_title = self.model().horizontalHeaderItem(column).text()
             item_col = self.model().itemFromIndex(idx.sibling(idx.row(), column))
-            column_data = item_col.text().strip()
+            data = item_col.data()
```
Comment (SomberNight) [inline electrum/gui/qt/util.py]: The name `data` is not very descriptive, I suggest renaming to `clipboard_data`.
I would rather not use `QStandardItem.setData()` and `QStandardItem.data()` without specifying the `role` argument.
At the top of this class, `MyTreeView`, let's define an attribute (of the class), e.g.:
`ROLE_CLIPBOARD_DATA = Qt.UserRole + 100`
Then, we could have:
```suggestion
            clipboard_data = item_col.data(self.ROLE_CLIPBOARD_DATA)
```

File: electrum/gui/qt/utxo_list.py [hunk 2]
```diff
@@ -96,6 +96,8 @@ def insert_utxo(self, idx, utxo: PartialTxInput):
         amount = self.parent.format_amount(utxo.value_sats(), whitespaces=True)
         labels = [name_short, address, label, amount, '%d'%height]
         utxo_item = [QStandardItem(x) for x in labels]
+        # Storing the full outpoint as the data payload of this item
+        utxo_item[0].setData(name)
```
Comment (SomberNight) [inline electrum/gui/qt/utxo_list.py]: There are already a few similar lines below, in particular `utxo_item[self.Columns.ADDRESS].setData(name, Qt.UserRole)`.
Let's move this below that one.
Also, like those lines below, please do not use a magic number here (`0`).
Along with the suggestion for the other file, we could have:
```suggestion
        utxo_item[self.Columns.OUTPOINT].setData(name, self.ROLE_CLIPBOARD_DATA)
```

File: electrum/gui/qt/util.py (hunk 1 above)
Comment (bilthon) [reply electrum/gui/qt/util.py]: Makes sense, done.

File: electrum/gui/qt/utxo_list.py (hunk 2 above)
Comment (bilthon) [reply electrum/gui/qt/utxo_list.py]: Also done.

**Comment by SomberNight** [comment]
Thanks

**Pull request merged**